### DISCUSSION TITLE
Make error

### DISCUSSION THREAD

**Discussion by ghost** [hyperledger/fabric#2383] [Q&A]
Hello all,
             I am building hyperledger fabric on my local machine using $make command. I have installed and set the path for GO. yet I still get an error.
 ll files are excluded from having license headers
Building github.com/client9/misspell/cmd/misspell -> misspell
Checking changed go files for spelling errors ...
xargs: misspell: No such file or directory
spell checker passed
Checking for go:generate parent path references
Checking trailing spaces ...
DEP: Checking for dependency issues..
./scripts/check_deps.sh
Building golang.org/x/tools/cmd/goimports -> goimports
Building mvdan.cc/gofumpt -> gofumpt
Building honnef.co/go/tools/cmd/staticcheck -> staticcheck
LINT: Running code checks..
./scripts/golinter.sh
Checking with goimports
./scripts/golinter.sh: line 19: goimports: command not found
make: *** [Makefile:179: linter] Error 127

**Comment by guoger** [comment]
there are two thing you probably could check:
1. see if `$GOPATH/bin` is in your `$PATH`, so executables in it could be found
2. `goimports` is properly installed into your `$GOPATH/bin` (cd into that dir and see if it exits)

**Comment by C0rWin** [comment]
Try to run `make gotools`, ensuring and installing all tools needed for the project support.
